source/sass/blocks/form-input-font.scss

```
@@ -0,0 +1,7 @@
+.form-input-font {
+  font-family: "Open Sans", "Helvetica", sans-serif;
```

А еще можно написать вот так:
```suggestion
  font-family: inherit;
```

Тогда браузерные стили перепишет правило "наследуйся" и не нужно будет в разных местах прописывать полное семейство